src/test/java/org/broadinstitute/hellbender/engine/spark/datasources/VariantsSparkSourceUnitTest.java

```diff
@@ -55,6 +56,29 @@ public void pairReadsAndVariantsTest_variantContexts(String vcf) {
         assertEqualVariants(getSerialVariantContexts(vcf), rddParallelVariantContexts.collect());
     }
 
+    @DataProvider(name = "loadMultipleVariants")
+    public Object[][] loadMultipleVariants() {
+        return new Object[][]{
+                {Arrays.asList(hg19_chr1_1M_dbSNP, hg19_chr1_1M_dbSNP_modified)},
+        };
+    }
+
+    @Test(dataProvider = "loadMultipleVariants", groups = "spark")
+    public void getMultipleParallelVariantsTest(List<String> vcfList) {
+        JavaSparkContext ctx = SparkContextFactory.getTestSparkContext();
+        VariantsSparkSource variantsSparkSource = new VariantsSparkSource(ctx);
+
+        JavaRDD<GATKVariant> rddParallelVariants =
+                variantsSparkSource.getParallelVariants(vcfList);
+
+        JavaRDD<GATKVariant> rddUnionVariants = vcfList.stream()
+                .map(vcf -> variantsSparkSource.getParallelVariantContexts(vcf))
+                .reduce(ctx.emptyRDD(), (result, rdd) -> result.union(rdd))
+                .map(vc -> VariantContextVariantAdapter.sparkVariantAdapter(vc));
```

Comment:
This test is basically duplicating the code being tested in the test itself... I recommend that you instead use `VariantsSource.getVariantsList()` to generate the expected List of variants, and compare that List with the result of `variantsSparkSource.getParallelVariants()` using apache's `CollectionUtils.isEqualCollection()` to compare ignoring the ordering of the variants.